modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -108,6 +108,9 @@ class {{classname}}(object):
 {{#isEnum}}
 {{#isContainer}}
         allowed_values = [{{#allowableValues}}{{#values}}{{#items.isString}}"{{/items.isString}}{{{this}}}{{#items.isString}}"{{/items.isString}}{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]  # noqa: E501
```

Comment:
Why not instead, in each allowed_values definitions add something like?
allowed_values = [{{#isNullable}}None, {{/isNullable}}{{#allowableValues}}{{#values}}{{#items.isString}}"{{/items.isString}}{{{this}}}{{#items.isString}}"{{/items.isString}}{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]  # noqa: E501

Reply:
Yeah, would probably be cleaner. I wasn't sure, feared to add too much commas or something :-)